submissions/rocket-silo/rocket-silo.md

```diff
@@ -0,0 +1,21 @@
+
+### Productivity Bonus Interlude
+
+When I say crafting operation, I refer to the main bar within the assembling-machine UI filling up to completion. For the sake of explanation, productivity is basically a multiplier on your output for your given input. The way this works is by taking 100 inputs and multiplying that by the percentage bonus from the modules within the machine. So, for example, take 4 productivity module 3s inside some copper cable assembling machine 3.
+The total bonus from the modules is 40%, so 10 copper plates becomes 20*(1+0.4)=28 copper cables. We can calculate the required input for a given output by the reverse of the process. So if we wanted 140 copper cable made with the influence of 4 productivity module 3s, we first divide by the amount per craft (2) to obtain we need 70 crafting operations we need including those from productivity. Then we divide 70 by the productivity multiplier of 1.4, which will sort out the 20 free operations and leave us with 50 crafting operations.
+
+### The Rocket
+
+The space rocket itself is the main reason why you place the silo down. Each rocket costs 49,100 iron, 92,500 copper, 9,500 coal, and 277,778 crude oil (https://factoriolab.github.io/list?p=rocket-part*100&v=1). This is added onto whatever you throw into the payload slot, but that can vary greatly depending on what you feed it. For most recipes, productivity modules pay off extremely slowly. For recipes involving a lot of raw materials, it pays off fast. Especially when used with the silo. 1 productivity module 3 in the silo itself drops the cost of each rocket to 10/11 or 90.91% of its original cost. Even a 9.09% drop of cost grants 4,463.6 iron, 8,409.1 copper, 863.6 coal, and 25,252.5 crude oil on average (https://factoriolab.github.io/list?z=eJwrcM7SMjQwUCtyLtDScqvzAkG1MmMAUBsG5Q__). That's a bit over 4 modules' worth of raw materials. What about all 4 modules being productivity module 3s? That yields a 28.57% drop of input needed. When applied to a rocket, that's 14,028 iron; 26,428 copper; 2,714 coal; and 79,365 crude oil saved by doing that, or just under 13 modules' worth of material saved (https://factoriolab.github.io/list?z=eJwrcA7UMjQxMLLQ0nJKUStyLtDScqsDQ7UyYwBzOAgo). In other words, its pays for itself three times over!
```

Comment:
This conclusion is super intersting! I think its definitely a cool fact that people should know about so giving it a bit more explanation could be nice. Again this paragraph feels a little listy so some refinement could be use there.

Reply:
Maybe a so long and thanks for all the fish joke at the end here :p